Storage/Rust/RustLogins.swift

```diff
@@ -850,6 +850,12 @@ public class RustLogins {
         return deferred
     }
 
+    public func registerWithSyncManager() {
+        queue.async {
```

Comment:
If you think the self will ever be `nil` then use weak here

```suggestion
        queue.async { [weak self] in
```

If you think that `self` could never be nil the good to `use unowned self`
```suggestion
        queue.async { [unowned self] in
```

Since you are in an async closure its good to add self as part of either weak or unowned.

Reply:
You might need to tweak other areas of this method for `weak` vs `unowned` changes